Fixes for 2025.1 change log (#18116)
nvaccess/nvda: user_docs/en/changes.md

The Add-on Store's automatic update system has been improved, allowing you to select channels for automatic updates, and run automatic updates in the background.

> It's now easier to manually refresh an OCR recognition result and toggle automatic refresh, with new commands.

Reviewer: ```suggestion
New commands have been added to manually refresh an OCR result, and toggle periodically refreshing OCR results.
```

Author: Thanks for this re-wording.

Wouldn't this one be still better?
```suggestion
New commands have been added to manually refresh an OCR result, and to toggle periodical refresh of OCR results.
```

or, if not, this one:
```suggestion
New commands have been added to manually refresh an OCR result, and to toggle periodically refreshing OCR results.
```

>   * `nvwave.WasapiWavePlayer` has been renamed to `WavePlayer`.

Reviewer: ```suggestion
  * `nvwave.WasapiWavePlayer` has been renamed to `WavePlayer`.
  Additionally, the method signature of its `__init__` has changed as follows:
    * The `outputDevice` parameter should now only be passed string arguments.
    * The deprecated `closeWhenIdle` and `buffered` parameters have been removed.
```

  * `gui.settingsDialogs.AdvancedPanelControls.wasapiComboBox` has been removed.
  * The `WASAPI` key has been removed from the `audio` section of the config spec.
  * The configuration key `config.conf["speech"]["outputDevice"]` has been removed.
    It has been replaced by `config.conf["audio"]["outputDevice"]`, which stores a Windows core audio endpoint device ID. (#17547)

>   * The `outputDevice` parameter to `WavePlayer.__init__` (formerly `WasapiWavePlayer.__init__`) should now only be passed string arguments.
>   * The deprecated `closeWhenIdle` and `buffered` parameters to `WavePlayer.__init__` (formerly `WasapiWavePlayer.__init__`) have been removed.

Reviewer: ```suggestion
```
